_config/config.yml

```diff
@@ -3,5 +3,5 @@ Name: logviewerconfig
 ---
 SilverStripe\Core\Injector\Injector:
   Psr\Log\LoggerInterface:
-    calls:
-      AddDataObjectHandler: [ pushHandler, [ %$SilverLeague\LogViewer\Handler\DataObjectHandler ] ]
+  calls:
```

Comment:
```suggestion
   calls:
```

Reply:
I spot this bug and it will break when run `composer dev-build`. `calls` should be 1 indentation further.